ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -99,12 +107,16 @@ open class ContactView: UIView {
             constraints.append(contentsOf: identifierLayoutConstraints())
         }
         titleLabel.translatesAutoresizingMaskIntoConstraints = false
-        labelContainer.addSubview(titleLabel)
+
+        pressedStateOverlay.translatesAutoresizingMaskIntoConstraints = false
+        avatarView.addSubview(pressedStateOverlay)
```

Comment:
We shouldn't add the pressedStateOverlay to the view hierarchy unless an action+target is set on the ContactView to handle the tap events. In other words, the overlay tap animation shouldn't be shown unless there the tap event has a handler.

Reply:
Aren't the `touchesBegan`, `touchesMoved`, `touchesEnded` functions considered as targets or are you saying we should check if those methods have been implemented before adding `pressedStateOverlay` to the view hierarchy?

Reply:
I think ContactView is missing something. If we want ContactView to be clickable, you need to expose an action selector (SEL) and a target (id). See the UIButton API. Example:
button.addTarget(self, action: action, for: .touchUpInside)
You don't need the event type param since you set the touch behavior in ContactView.

If you don't set a target/selector, then the ContactView should not have a touch down behavior at all since it's not clickable.

Reply:
As we discussed, you will need to address this in a subsequent change.